corehq/apps/geospatial/forms.py

```diff
@@ -20,20 +18,14 @@ class GeospatialConfigForm(forms.ModelForm):
     class Meta:
         model = GeoConfig
         fields = [
-            "location_data_source",
             "user_location_property_name",
             "case_location_property_name"
         ]
 
-    location_data_source = forms.CharField(
-        label=_("Fetch user location data from"),
-        widget=forms.Select(choices=LOCATION_SOURCE_OPTIONS),
-        required=False,
-    )
     user_location_property_name = forms.CharField(
-        label=_("Custom user field name"),
-        required=False,
-        help_text=_("The name of the user field which stores the users' geo-location data."),
+        label=_("Fetch mobile worker location data from custom field"),
+        required=True,
+        help_text=_("The name of the mobile worker custom field which stores the users' geo-location data."),
```

Comment:
How about?
```
The name of the field that stores the users' geo-location data.
```

Reply:
@SmittieC The question is whether it's obvious to the user _which_ field that is, hence I thought saying explicitly it's the _mobile worker custom field_ might be good.